.circleci/scripts/binary_populate_env.sh

```diff
@@ -78,6 +71,20 @@ fi
 
 export PYTORCH_BUILD_NUMBER=1
 
+# Here PYTORCH_EXTRA_INSTALL_REQUIREMENTS is already set for the all the wheel builds hence append TRITON_CONSTRAINT
+if [[ -n "${PYTORCH_EXTRA_INSTALL_REQUIREMENTS:-}" ]]; then
```

Comment:
Could you remind me again why this logic is duplicated with builder https://github.com/pytorch/builder/blob/c9f9aae0bc24db05f0e325b025fcdbb3c4e6d265/manywheel/build_cuda.sh#L268-L281?  As both will be called in https://github.com/pytorch/pytorch/blob/3e8b56d362ace0eaa5007e69b431b7ba3f75be46/.github/workflows/_binary-build-linux.yml#L249-L254 from what I see

Reply:
Ah ok, I see it that the builder logic is removed in https://github.com/pytorch/builder/pull/1712